modules/core/src/main/scala/fs2/kafka/ProducerRecords.scala

```diff
@@ -6,73 +6,20 @@
 
 package fs2.kafka
 
-import cats.{Show, Traverse}
-import cats.syntax.show._
 import fs2.Chunk
-import fs2.kafka.internal.syntax._
 
 import scala.collection.mutable
-
-/**
-  * [[ProducerRecords]] represents zero or more `ProducerRecord`s,
-  * together with an arbitrary passthrough value, all of which can
-  * be used with [[KafkaProducer]]. [[ProducerRecords]]s can be
-  * created using one of the following options.<br>
-  * <br>
-  * - `ProducerRecords#apply` to produce zero or more records
-  * and then emit a [[ProducerResult]] with the results and
-  * specified passthrough value.<br>
-  * - `ProducerRecords#one` to produce exactly one record and
-  * then emit a [[ProducerResult]] with the result and specified
-  * passthrough value.<br>
-  * <br>
-  * The [[passthrough]] and [[records]] can be retrieved from an
-  * existing [[ProducerRecords]] instance.<br>
-  */
-sealed abstract class ProducerRecords[+P, +K, +V] {
-
-  /** The records to produce. Can be empty for passthrough-only. */
-  def records: Chunk[ProducerRecord[K, V]]
-
-  /** The passthrough to emit once all [[records]] have been produced. */
-  def passthrough: P
-}
+import cats.Traverse
 
 object ProducerRecords {
```

Comment:
Maybe it's worth to move this companion to the type alias definition to improve discoverability? Currently type itself defined in one place and its component in another.

Reply:
Sure, I don’t want to bloat the package object file too much but I guess that’s ok for just a few constructors